.github/workflows/sqlite3.yml

```diff
@@ -39,12 +44,3 @@ jobs:
 
     - name: Run Rust tests with Wasm
       run: make rusttestwasm
-
-    - name: reconfigure with Wasm in WasmEdge mode
-      run: make clean && ./configure --enable-wasm-runtime-wasmedge
-
-    - name: Run tests
-      run: make test
-
-    - name: Run tests with Wasm
-      run: make rusttestwasm
```

Comment:
why is this going?

Reply:
I talked with @psarna this morning, we don't need wasmedge right now and the first pass covers enough for all our C code. We were duplicating a lot of work.